client/src/main.c

```diff
@@ -583,6 +583,30 @@ void spiceClipboardRequest(const SpiceDataType type)
     state.lgc->request(spice_type_to_clipboard_type(type));
 }
 
+// set the compositor hint to bypass for low latency
+void bypassCompositor(unsigned long value)
```

Comment:
this should be `static void` as it's only used locally, and please use a boolean, not unsigned long.